src/screens/ChannelListScreen/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-use-before-define */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Disabling eslint rules at the top of your file is generally not a good practice. It's better to address the issues that these rules are designed to catch. If you absolutely must disable a rule, it's better to do so on a per-line basis so it's clear why the rule is being disabled.

```diff
- /* eslint-disable @typescript-eslint/no-explicit-any */
- /* eslint-disable no-use-before-define */
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->